src/renderer/containers/root.js

```diff
@@ -8,6 +8,13 @@ export default class Root extends PureComponent {
     store: PropTypes.object.isRequired
   };
 
+  componentDidMount() {
+    console.warn(`Attention!
```

Comment:
Also can we add a check to show this only in production? What do you think @perry-mitchell? 

```js
if (process.env.NODE_ENV === "production") {
```

Reply:
Sure! I don’t think it’s the worst if it’s everywhere.. but no harm either way really.

Reply:
@perry-mitchell you're right. its fine if it's everywhere